test/mocha/10-meters.js

```diff
@@ -70,6 +91,25 @@ describe('meters', () => {
       _get2.meter.should.not.deep.equal(_get.meter);
       _get2.meter.controller.should.equal('c2');
     });
+    it('should throw error when updating meter that does not exist in database',
+      async () => {
+        const meterNotInDB = 'zH2TzRv5Qs9SsngkYSWuU1x';
+        const meter = {id: meterNotInDB, sequence: 0};
+
+        let err;
+        let res;
+        try {
+          res = await meters.update({meter});
+        } catch(e) {
+          err = e;
+        }
+        should.exist(err);
+        should.not.exist(res);
+        err.name.should.equal('InvalidStateError');
+        err.message.should.equal('Could not update configuration. ' +
```

Comment:
I'm not so sure we should be testing specific error message text. Do we lose code coverage somehow if we don't test this? I can see these messages changing, particularly ones from dependencies like `assert-plus` -- and then our test suites will fail. One shouldn't make code paths depend on error messages, only error types anyway.

Reply:
@dlongley I can remove the tests for `assert-plus` assertions, we'll be at 91% so that'll be fine. As for whether or not we should assert error message texts I am not sure either, should I remove it?

Reply:
@dlongley oh I misunderstood, you mean I should just remove the assertions on the err message but keep the tests, correct?